Docs/Configuration.tex

```diff
@@ -5170,15 +5170,15 @@ \subsection{Other Variables}\label{nvramvarsother}
 
   \begin{itemize}
   \item \texttt{boot-save-log=VALUE} --- debug log save mode for normal boot.
-  	\begin{itemize}
+    \begin{itemize}
```

Comment:
All the whitespace cleanups look fine and useful to me.

Reply:
I've pushed the whitespace handling changes - tyvm. (If you want to rebase your PR branch onto master, it should automatically drop that commit.)